### PR TITLE
[Translation] Added support for multiple files or directories in XliffLintCommand

### PR DESCRIPTION
**Pull request by yceruto**
See https://github.com/symfony/symfony/pull/28522

### CODE REVIEW

File: translation/lint.rst [hunk 1]
```diff
@@ -16,11 +16,15 @@ translation file using the ``lint:yaml`` and ``lint:xliff`` commands:
 
     # lint a single file
     $ ./bin/console lint:yaml translations/messages.en.yaml
-    $ ./bin/console lint:xliff translations/messages.en.xlf
+    $ ./bin/console lint:xliff translations/messages.en.xlf
```
Comment (xabbuh) [inline translation/lint.rst]: I think we should revert this change :)

File: translation/lint.rst [hunk 2]
```diff
@@ -16,11 +16,15 @@ translation file using the ``lint:yaml`` and ``lint:xliff`` commands:
...
     # lint a single file
     $ ./bin/console lint:yaml translations/messages.en.yaml
-    $ ./bin/console lint:xliff translations/messages.en.xlf
+    $ ./bin/console lint:xliff translations/messages.en.xlf   
 
     # lint a whole directory
     $ ./bin/console lint:yaml translations
     $ ./bin/console lint:xliff translations
+    
+    # lint a multiple files or directories
```
Comment (xabbuh) [inline translation/lint.rst]: lint multiple [...]

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Yonel. This is a nice improvement to the command.
